src/main/java/sirius/db/mixing/query/BaseQuery.java

```diff
@@ -140,6 +164,9 @@ public void iterateAll(Consumer<E> consumer) {
      */
     public List<E> queryList() {
         List<E> result = new ArrayList<>();
+        if (forceFail) {
+            return result;
```

Comment:
```suggestion
            return Collections.emptyList();
```
ist ein bisschen expliziter, und kann nicht aus versehen per re-ordering kaputt gehen